packages/bpk-component-barchart/src/BpkBarchartBar.js

```diff
@@ -51,35 +51,53 @@ const BpkBarchartBar = (props) => {
     onClick,
     outlier,
     selected,
+    padding,
```

Comment:
`BpkBarchartBar` isn't exported, but `BpkBarchartBars` is (see https://github.com/Skyscanner/backpack/blob/master/packages/bpk-component-barchart/index.js#L20).

I think it's too early to make a decision with all the other components. This will become clearer once a lien chart gets added into the mix.